lib/rocket_chat/messages/room.rb

```diff
@@ -309,7 +309,7 @@ def members(room_id: nil, name: nil, offset: nil, count: nil, sort: nil)
       attr_reader :session
 
       def room_option_hash(options)
-        args = [options, :members, :read_only, :custom_fields]
+        args = [options, :members, :read_only, :custom_fields, :extra_data]
```

Comment:
Change looks good, but could you update the documentation too? See docs/channels.md#L17

Reply:
Actually, don't worry.. I'll update it when I bump the release 😉